packages/langium/src/index/ast-descriptions.ts

```diff
@@ -23,10 +23,12 @@ export interface ReferenceDescription {
     targetUri: URI
     /** how to find target AstNode inside the document */
     targetPath: string
-    /** start offset of the reference text */
-    start: number
-    /** end offset of the reference text */
-    end: number
+    /** start position of the reference text */
+    start: {line: number, character: number}
+    /** end position of the reference text */
+    end:  {line: number, character: number}
```

Comment:
Shouldn't we keep the offset-based positions like in Xtext? Why did you change this to line/character positions?